iib/workers/tasks/utils.py

```diff
@@ -195,6 +195,14 @@ class RequestConfig:
         identify the appropriate ``binary_image`` to use.
     """
 
+    # these attrs should not be printed out
+    _secret_attrs = [
+        'cnr_token',
```

Comment:
This is deprecated

Reply:
and this part should be taken care of in web

Reply:
This it is for worker, in web it is different.

Reply:
By deprecated I understand as `works but you shouldn't use` instead of `don't work anymore` right?

If so the commit is ok because it might have "someone" that still uses it and we don't want to print a secret.

However, if the feature is completely removed and sending this token will end up in a rejection then I agree we don't need this part.

FWIW It's better to be prepared for the worst scenario than not having it and "someone" still using deprecated stuff and leaking secrets.

Reply:
resolved by discussion